build/mscore.desktop.in

```diff
@@ -1,14 +1,10 @@
 [Desktop Entry]
-Version=2.0
-Comment=Create, play and print sheet music
-Comment[ru]=Визуальный редактор нотных партитур
-Comment[fr]=Gravure de partitions musicales
-Exec=mscore %F
+Version=1.1@This_is_version_of_Desktop_Entry_Specification@@It_is_NOT_MuseScore_version@
+Name=@MUSESCORE_NAME_VERSION@@Variables_substituted_by_CMAKE_on_installation@
 GenericName=Music notation
-GenericName[de]=Notensatz
-GenericName[fr]=Notation musicale
-Icon=mscore
-Name=MuseScore
+Comment=Create, play and print sheet music
```

Comment:
That was accidental. Files with the ".desktop" extension are special on Linux and something strange happened when I changed the extension to ".in". I corrected most of the changes but I didn't notice the missing translations.